modules/developing/pages/astream-cdc.adoc

```diff
@@ -13,7 +13,8 @@ This doc will show you how to create a CDC connector for your Astra DB deploymen
 
 [NOTE]
 ====
-Enabling CDC for {astra_db} will result in increased costs based on your {astra_stream} usage. See xref:operations:astream-pricing.adoc[Astra Streaming Pricing] and the https://www.datastax.com/products/datastax-astra/cdc-for-astra-db[Astra CDC pricing page].
+Enabling CDC for {astra_db} will result in increased costs based on your {astra_stream} usage.
+See https://www.datastax.com/products/astra-streaming/pricing[Astra Streaming pricing^]{external-link-icon}.
```

Comment:
We don't actually mention CDC here. Can you ask Martin Carrig if this is available somewhere?

Reply:
@jgillenwater @msmygit the metering rates are here, reverted the link
https://www.datastax.com/products/datastax-astra/cdc-for-astra-db